docs/docs/1.0.7/policies/protocol-support-in-kuma.md

```diff
@@ -14,7 +14,7 @@ By doing this,
 
 :::: tabs :options="{ useUrlFragment: false }"
 ::: tab "Kubernetes"
-On `Kubernetes`, to give Kuma a hint that your service supports `HTTP` protocol, you need to add a `<port>.service.kuma.io/protocol` annotation to the `k8s` `Service` object.
+On `Kubernetes`, to give Kuma a hint that your service supports `HTTP` protocol, you need to add a `<port>.service.kuma.io/protocol` annotation to the `k8s` `Service` object or use `appProtocol` of the `k8s` `Service` object.
```

Comment:
please provide tabs `Kubernetes < 1.18`, `Kubernetes 1.18+`. I think mixing both approaches might be confusing for users